init.sh

```diff
@@ -1,4 +1,8 @@
-KEY="mykey"
+#!/bin/bash
+KEYS[0]="mykey"
+KEYS[1]="dev1"
+KEYS[2]="dev2"
+KEYS[3]="dev3"
```

Comment:
nit, three keys in total should be enough to showcases that you can add multiple keys